src/index.html

```diff
@@ -8,14 +8,16 @@
   </head>
   <body>
     <table class="dashboard">
-      <tr>
-        <th>Name</th>
-        <th>Gender</th>
-        <th>Born</th>
-        <th>Died</th>
-        <th>Age</th>
-        <th>Century</th>
-      </tr>
+      <tbody>
```

Comment:
seems like it's should be `thead`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead